NC-NEWS/src/components/Articles.jsx

```diff
@@ -0,0 +1,26 @@
+import getArticles from "../api";
+import ArticlesCard from "./ArticlesCard";
+import { useEffect, useState } from "react";
+
+const Articles = () => {
+  const [articlesData, setArticlesData] = useState([]);
+
+  useEffect(() => {
+    getArticles().then((articles) => {
+      setArticlesData(articles);
+    });
+  }, []);
+
+  return (
+    <>
+      <h2>All Article's</h2>
+      <ul>
+        {articlesData.map((article) => {
+          return <ArticlesCard key={article.title} article={article} />;
+        })}
+      </ul>
```

Comment:
Excellent, good to see you're extracting the rendering of an individual article into its own component using props, well done.